packages/design-system/src/components/SearchBar/style.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../scss/variables/colors';
+
+$base-class: "search-bar";
+
+.#{$base-class}__container {
+  display: inline-flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  height: 32px;
+  width: 100%;
+}
+
+.#{$base-class}__icon {
+  position: absolute;
+  align-self: center;
+
+  &--search {
+    @extend .#{$base-class}__icon;
+    left: 0;
+    padding: 7px 13px;
+    fill: $icon-primary-color;
+
+    &-compact {
+      cursor: pointer;
+    }
+  }
+
+  &--clear {
+    @extend .#{$base-class}__icon;
```

Comment:
no need for extend

Reply:
Hmm without extend my styles are breaking 😕

Reply:
that is because you used only one class in jsx: className={styles[`${baseClass}__icon--clear`]} instead of className={cx(styles[`${baseClass}__icon`], styles[`${baseClass}__icon--clear`])}